COMPATIBILITY.md

```diff
@@ -0,0 +1,6 @@
+# Compatibility 
+## hydra-genetics/prealignment
+
+| prealignment version | alignment version |
+| --- | --- |
+| v0.2.0 |  |
```

Comment:
This one will be updated as soon as we have a release right? But how do we do that? It needs to be set the second we release kind of.

Reply:
We will have a procedure where one updates it manual before doing the release, at least until we find a automatic way of doing it

Reply:
ok 👍